app/services/update_school_devices_service.rb

```diff
@@ -32,20 +35,41 @@ def notify_other_agents
     CapUpdateNotificationsService.new(*allocation_ids, notify_school: notify_school).call if allocation_ids.any?
   end
 
+  def record_allocation_change_meta_data(allocation_id:, prev_allocation:, new_allocation:)
+    if allocation_change_category || allocation_change_description
```

Comment:
A terminating `unless` statement would be preferable to wrapping this in an `if` block.

Reply:
The thing is that Rubocop doesn´t like inline `if` or `unless` clauses when the statement is a multiline one.